test/Performance/run-perf_tests.py

```diff
@@ -0,0 +1,792 @@
+def get_sha1_file( filename ):
+    """
+    This module generates sha1sum for the given file
+    Args:
+        filename (str): Full path to the file
+    Returns:
+        sha1sum hash of the file
+    """
+    hash_size = 256
+    sha1 = hashlib.sha1()
+    with open( filename, 'rb' ) as f:
+        while True:
+            data = f.read( hash_size )
+            if not data:
+                break
+            sha1.update( data )
+    return "0x" + sha1.hexdigest()
+       
+def insert_server_entry( conn, server_name, server_version ):
+    """
+    This module inserts a new entry into Servers table
+    Args:
+        conn (obj): Connection object to the Results database
+        server_name (str): Name of the Test Server that the tests are run against
+        server_version (str): @@Version of the Test Server
+    Returns:
+        N/A
+    """
+    query = "INSERT INTO Servers ( HostName, Version ) VALUES ( '{0}', '{1}' )"
+    cursor = conn.cursor()
+    cursor.execute( query.format( server_name, server_version ))
+    cursor.close()
+ 
+def insert_client_entry ( conn, name ):
+    """
+    This module inserts a new entry into Clients table
+    Args:
+        conn (obj): Connection object to the Results database
+        name (str): Name of the Client machine that the tests are run on
+    Returns:
+        N/A
+    """
+    query = "INSERT INTO Clients ( HostName ) VALUES( '{0}' )"
+    cursor = conn.cursor()
+    cursor.execute( query.format( name ))
+    cursor.close()
+ 
+def insert_team_entry ( conn, name ):
+    """
+    This module inserts a new entry into Teams table
+    Args:
+        conn (obj): Connection object to the Results database
+        name (str): Team name
+    Returns:
+        N/A
+    """
+    query = "INSERT INTO Teams ( TeamName ) VALUES( '{0}' )"
+    cursor = conn.cursor()
+    cursor.execute( query.format( name ))
+    cursor.close()
+ 
+def insert_test_entry( conn, name ):
+    """
+    This module inserts a new entry into PerformanceTests table
+    Args:
+        conn (obj): Connection object to the Results database
+        name (str): Test name
+    Returns:
+        N/A    
+    """
+    #TO-DO Remove unnecessary columns from the table and fix the query string. Amd64 and 0 are used to bypass not null
+    query = "INSERT INTO PerformanceTests ( TestName, Arch, HashVer ) VALUES( '{0}', 'Amd64', 0 )"
+    cursor = conn.cursor()
+    cursor.execute( query.format( name ))
+    cursor.close()
+ 
+def insert_driver_entry( conn, driver_path, driver_hash ):
+    """
+    This module inserts a new entry into Drivers table
+    Args:
+        conn (obj): Connection object to the Results database
+        name (str): Full path to the driver
+        driver_hash (bin): sha1sum hash of the driver 
+    Returns:
+        N/A    
+    """
+    file_date = time.strftime( fmt, time.gmtime( os.path.getmtime( driver_path )))
+    query = "INSERT INTO Drivers ( Arch, FileDate, SHA1, HashVer ) VALUES ( ?, ?, {0}, 1 )"

```

Comment:
What is HashVer? Is it because sha**1** was used? If that is the case, I think it's better to make the HashVer a global variable so we can change it whenever a different sha version is used.

Also, is there a reason why you used bind param here? While for most of the other cases you just use query.format(...).